src/main/java/org/tron/common/runtime/vm/PrecompiledContracts.java

```diff
@@ -1045,10 +1072,19 @@ public long getEnergyForData(byte[] data) {
 
       final List<Actuator> actuatorList = ActuatorFactory
           .createActuator(trx, getDeposit().getDbManager());
+      Optional<List<Actuator>> optionalList = Optional.ofNullable(actuatorList);
+      ProposalCreateActuator proposalCreateActuator;
       try {
-        actuatorList.get(0).validate();
-        actuatorList.get(0).execute(getResult().getRet());
-        id = getDeposit().getDbManager().getDynamicPropertiesStore().getLatestProposalNum();
+        if (!optionalList.isPresent() || optionalList.get().isEmpty()){
+          throw new ContractExeException("can't initiate ProposalCreateActuator for precompiled vm method");
+        }
+        else{
+          proposalCreateActuator = (ProposalCreateActuator)optionalList.get().stream().findFirst().get();
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Call "Optional#isPresent()" before accessing the value. ([squid:S3655](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS3655))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1420&issues=AWWTWS-S7khxiKmGgQPl&open=AWWTWS-S7khxiKmGgQPl)

[ISSUE_KEY]: AWWTWS-S7khxiKmGgQPl